tests/averaging/rotation/test_shonan.py

```diff
@@ -119,6 +119,37 @@ def test_pickleable(self):
         except TypeError:
             self.fail("Cannot dump rotation averaging object using pickle")
 
+    def test_nonconsecutive_indices(self):
+        """Run rotation averaging on a graph with indices that are not ordered as [0,...,N-1].
+
+        Note: Test would fail if Shonan keys were not temporarily re-ordered inside the implementation.
+        See https://github.com/borglab/gtsam/issues/784
+        """
+        num_images = 4
+
+        wTi0 = Pose3(Rot3(), np.array([1, 1, 0]))
```

Comment:
this is unused

Reply:
good point -- I moved it to an inline comment now